script.js

```diff
@@ -3,12 +3,15 @@ document.addEventListener('DOMContentLoaded', function () {
 
     const btnEnviar = document.querySelector('button');
     const radios = document.querySelectorAll('input[name="voto"]');
+    const mensagemErro = document.getElementById('mensagemErro');
     function verificarSelecao() {
         const Selecionada = Array.from(radios).some(radio => radio.checked);
         if (Selecionada) {
             btnEnviar.disabled = false;
+            mensagemErro.style.display = 'none'; // Esconde a mensagem de erro quando uma opção é selecionada
```

Comment:
Trocar a lógica para adicionar/remover a classe `is-hidden`.

Reply:
check